powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationServiceBehavior.java

```diff
@@ -2479,7 +2480,11 @@ public void expireActivations() {
         try (final Stream<ActivationRecordEntity> abandonedActivations = activationRepository.findAbandonedActivations(activationStatuses, lookBackTimestamp, currentTimestamp)) {
             abandonedActivations.forEach(activation -> {
                 logger.info("Removing abandoned activation with ID: {}", activation.getActivationId());
-                deactivatePendingActivation(currentTimestamp, activation, true);
+                try {
+                    deactivatePendingActivation(currentTimestamp, activation, false);
+                } catch (GenericServiceException e) {
+                    logger.error("Activation not found, activation ID: {}", activation.getActivationId());
```

Comment:
🤔 Is the log message correct? Should be something like `Activation was not possible to deactivate, activation ID: {}`?